doc/concepts/recipe.rst

```diff
@@ -1,22 +1,77 @@
 .. _modelrecipe:
 
 Recipes
-===============
+=======
 
 An Arbor *recipe* is a description of a model. The recipe is queried during the model
 building phase to provide information about cells in the model, such as:
 
-  * the number of cells in the model;
-  * the type of a cell;
-  * a description of a cell, e.g. with soma, synapses, detectors, stimuli;
-  * the number of spike targets;
-  * the number of spike sources;
-  * the number of gap junction sites;
-  * incoming network connections from other cells terminating on a cell;
-  * gap junction connections on a cell.
+  * The **number of cells** in the model.
+  * The **kind** of each cell.
+  * The **description** of each cell, e.g. with morphology, synapses, detectors, stimuli.
```

Comment:
I'm slightly confused as to what the **description** of a cell precisely refers to. Does it include spike targets, sources, etc? The [conceptual cable cell page](https://arbor-rtd-dev.readthedocs.io/en/latest/concepts/cable_cell.html) uses **description** in a broad fashion.

Reply:
I'll make this a bit clearer. But description is not really a vague term in arbor, it's the cell itself. And our recipe method is called `get_cell_description`. I use the word description in the example to link back to the recipe terminology.

Reply:
The term 'description' is a bit overloaded. Also, there is no matching `set_description`, right? Is it (with the new `cable_cell`) everything passed in the constructor?